core/src/main/java/org/apache/brooklyn/core/entity/AbstractEntity.java

```diff
@@ -196,7 +196,11 @@
             "entity.group.added", "Group dynamically added to entity");
     public static final BasicNotificationSensor<Group> GROUP_REMOVED = new BasicNotificationSensor<Group>(Group.class,
             "entity.group.removed", "Group dynamically removed from entity");
-    
+
+    public static final AttributeSensor<String> ENTITY_ID = Attributes.ENTITY_ID;
+    public static final AttributeSensor<String> APPLICATION_ID = Attributes.APPLICATION_ID;
+    public static final AttributeSensor<String> CATALOG_ID = Attributes.CATALOG_ID;
```

Comment:
simpler to handle the requirement that entity ID is the key in `MapAggregator` could be for entity ID to be the default key.  that feels like a sensible common default, and means we don't (yet) need to add these artificial sensors.